src/views/Migration/index.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState, useRef, useMemo } from 'react'
+import { useRouter } from 'next/router'
+import { useWeb3React } from '@web3-react/core'
+import { Heading, Text, Button, ArrowForwardIcon, Link } from '@pancakeswap/uikit'
+import { useTranslation } from 'contexts/Localization'
+import { usePollFarmsV1WithUserData } from 'state/farmsV1/hooks'
+import { VaultKey } from 'state/types'
+import { useFetchUserPools } from 'views/Migration/hook/V1/Pool/useFetchUserPools'
+import { useFetchPublicPoolsData } from 'views/Migration/hook/V1/Pool/useFetchPublicPoolsData'
+import PageHeader from 'components/PageHeader'
+import Page from 'components/Layout/Page'
+import ProgressSteps, { Step, ProgressStepsType } from './components/ProgressSteps'
+import MigrationSticky from './components/MigrationSticky'
+import OldPool from './components/MigrationStep1/OldPool'
+import OldFarm from './components/MigrationStep1/OldFarm'
+import NewPool from './components/MigrationStep2/NewPool'
+import NewFarm from './components/MigrationStep2/NewFarm'
+
+const MigrationPage: React.FC = () => {
+  const { t } = useTranslation()
+  const { account } = useWeb3React()
+  const tableWrapperEl = useRef<HTMLDivElement>(null)
+  const router = useRouter()
+  const [step, setStep] = useState<ProgressStepsType>(ProgressStepsType.STEP1)
+  const steps: Step[] = [
+    {
+      stepId: ProgressStepsType.STEP1,
+      canHover: true,
+      text: t('Unstake LP tokens and CAKE from the old MasterChef contract.'),
+    },
+    {
+      stepId: ProgressStepsType.STEP2,
+      canHover: true,
+      text: t('Stake LP tokens and CAKE to the new MasterChef v2 contract.'),
+    },
+  ]
+
+  // v1 Farms
+  usePollFarmsV1WithUserData(false)
+
+  // v1 Pools
+  useFetchPublicPoolsData()
```

Comment:
Because v1 Pools data is in hook. 
Every time from step2 back to step1. The pool will fetching again, but v1 farms data in store.
So v1 pool will always have loading animation, but v1 farms will not.